src/HeaderRow.js

```diff
@@ -26,9 +34,13 @@ var HeaderRow = React.createClass({
     height: PropTypes.number.isRequired,
     columns: PropTypes.arrayOf(React.PropTypes.shape(ExcelColumn)).isRequired,
     onColumnResize: PropTypes.func,
+    onSort: PropTypes.func.isRequired,
     style: PropTypes.shape(HeaderRowStyle)
+
   },
 
+  mixins: [ColumnUtilsMixin],
```

Comment:
I dont think it needs to be a mixin, no. It doesn't use any internal component attributes or lifecycle events so it can be a separate file